products/models.py

```diff
@@ -55,8 +68,33 @@ class Product(AuditTimeStamp):
     description = models.CharField(
         _("Product Description"), max_length=1000, blank=True
     )
+
+    image = models.ImageField(upload_to=image_path, default="default_image.png")
```

Comment:
This should be nullable.

Reply:
Done

Reply:
I do not understand this comment. When we have a placeholder image in case a product image is not given by the person who adds a product. Why should we make it nullable? Please let me know

Reply:
Let me brief: We never mend our database to follow the rules for something as much as a placeholder image. Let's suppose someone changes/asks you to change the listing of the products based on the image available or maybe someone asks you to change the placeholder image would you keep changing your database for that? or match image name string patterns for the product list ordering? **OR** The placeholder check should be on the front end of the context creation instead of saving lots of default_image.png? which even could be deleted at some point in time? What's your suggestion? Did you get what i was trying to say?

Reply:
I have made it nullable